modules/ipi-install-out-of-band-management.adoc

```diff
@@ -6,8 +6,18 @@
 [id="out-of-band-management_{context}"]
 = Out-of-band management
 
-Nodes will typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
+Nodes typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
 
 Each node must be accessible via out-of-band management. When using an out-of-band management network, the provisioner node requires access to the out-of-band management network for a successful {product-title} installation.
 
-The out-of-band management setup is out of scope for this document. We recommend setting up a separate management network for out-of-band management. However, using the provisioning network or the baremetal network are valid options.
+The out-of-band management setup is out of scope for this document. You can use a separate management network for out-of-band management. However, using the provisioning network or the baremetal network are also valid options.
+
+[NOTE]
+====
+You can use a maximum of two network interfaces on the bootstrap VM. If you configure a separate management network for out-of-band management, and you are using a provisioning network, the bootstrap VM requires routing access to the out-of-band network by using the bare-metal interface. In this scenario, the bootstrap VM can then access three networks:
```

Comment:
"You can use" may be misleading here. The user doesn't really get much say in how the installer creates the bootstrap VM. The bootstrap VM always has an interface on the baremetal network, and if the provisioning network is enabled then there will be an interface on that network too.

@cybertron is it accurate to say that the management network must be routable _from the baremetal network_ (i.e. being routable from the provisioning network doesn't count)? Because I don't think that's true, but if it were it should probably be considered a bug because if there is a provisioning network then making the management network routable via that rather than the baremetal network is almost always what you want.

BTW the same network is being referred to as the 'out-of-band network' here, but the 'management network' on l. 21. I think the latter is correct.

I would also replace 'by using the bare-metal interface' with 'from the baremetal network interface'.

Reply:
Thanks Zane, I updated the first sentence to remove "you can use"

@cybertron - I updated to "from the bare-metal interface" and kept "management network" consistent, thanks! 
Regarding your other point, I don't know if it's possible to make the management network routable from either the bare-metal or the provisioning network. I was just going off the feedback in the bug with these updates. I guess this needs to be verified by QE? And if it's possible to route through either interface, then promote using the provisioning interface?

Reply:
My understanding is that typically the provisioning network isn't routable so unless your BMCs are actually on the provisioning network (which seems problematic if it's not routable) then it would have to be the baremetal interface.

That said, it doesn't strictly matter. We could just say that the management network must be reachable from one of the two allowed interfaces on the bootstrap node.